src/main/java/controller/LadderController.java

```diff
@@ -0,0 +1,41 @@
+package controller;
+
+import domain.Ladder;
+import domain.Users;
+import view.InputView;
+import view.OutputView;
+
+public class LadderController {
+    private Users users;
+    private Ladder ladder;
+
+    public void run() {
+        generateUsers();
+        generateLadder();
+        printResult();
+    }
```

Comment:
```suggestion
    public void run() {
        Users users = generateUsers();
        Ladder ladder = generateLadder();
        printResult(users, ladder);
    }
```
얘네도 지역변수로 사용해보면 좋지 않을까요?